runtime/src/bank.rs

```diff
@@ -2021,7 +2026,7 @@ impl Bank {
             .unwrap_or_default()
     }
 
-    pub fn set_alpenglow_clock_sysvar(&self, unix_timestamp: UnixTimestamp) {
+    pub fn update_clock_from_footer(&self, unix_timestamp: UnixTimestamp) {
```

Comment:
might be clearer to use a different type for `UnixTimestamp` as the doc comment on this type specifies seconds.